src/relay/client.js

```diff
@@ -271,6 +271,37 @@ export class RelayClient {
     })
   }
 
+  async emptySelfSend (transaction) {
```

Comment:
This is clever, but I'm on the fence about it. Partially for the reasons we discussed out-of-band, and because it is unencrypted and sent to the relay servers.  It's true that they generally know where messages end up going, but we should minimize what they do know as much as possibhle.